### PR TITLE
XML Codestyle for the file application.xml

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

XML Codestyle for the file `administrator/components/com_config/model/form/application.xml`
### Testing Instructions
- Apply this patch
- go to the global config
- confirm nothing is broken there
### Documentation Changes Required

none

### CODE REVIEW

**Review by andrepereiradasilva — CHANGES_REQUESTED** [review]
if we are doing a full cs review of the xml you should folllow the name, type, labeldescription order as code style rules.

File: administrator/components/com_config/model/form/application.xml [hunk 1]
```diff
@@ -150,7 +162,8 @@
 			label="COM_CONFIG_FIELD_REDIS_PERSISTENT_LABEL"
 			description="COM_CONFIG_FIELD_REDIS_PERSISTENT_DESC"
 			filter="integer"
-			showon="caching:1,2[AND]cache_handler:redis">
+			showon="caching:1,2[AND]cache_handler:redis"
+		>
```
Comment (andrepereiradasilva) [inline administrator/components/com_config/model/form/application.xml]: add a tab here

**Comment by yvesh** [comment]
I have tested this item :white_check_mark: successfully on c3673d28abafe04d835070ef84377f5cd3b608f3<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12080">issues.joomla.org/joomla-cms/12080</a>.</sub>

**Review by wojsmol — CHANGES_REQUESTED** [review]
See https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

**Comment by wojsmol** [comment]
@zero-24 @andrepereiradasilva  Please see https://github.com/zero-24/joomla-cms/issues/17

**Comment by zero-24** [comment]
Looks good just one smal comment in that PR. Thanks @wojsmol :+1:

File: administrator/components/com_config/model/form/application.xml [hunk 2]
```diff
@@ -1003,49 +1077,68 @@
...
-				name="core.edit.state"
-				title="JACTION_EDITSTATE"
-				description="COM_CONFIG_ACTION_EDITSTATE_DESC" />
-			<action
-				name="core.edit.own"
-				title="JACTION_EDITOWN"
-				description="COM_CONFIG_ACTION_EDITOWN_DESC" />
+			filter="rules"
+			>
+				<action
```
Comment (andrepereiradasilva) [inline administrator/components/com_config/model/form/application.xml]: why are the  `action` idented with an extra tab? the `option` are not...

**Review by andrepereiradasilva — CHANGES_REQUESTED** [review]

File: administrator/components/com_config/model/form/application.xml (hunk 2 above)
Comment (wojsmol) [inline administrator/components/com_config/model/form/application.xml]: fixed in https://github.com/joomla/joomla-cms/pull/12080/commits/351db4e4cbe3ca21087213febe9c39a5c0ab8992

**Comment by zero-24** [comment]
Thanks @wojsmol

**Review by andrepereiradasilva — APPROVED** [review]

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 351db4e4cbe3ca21087213febe9c39a5c0ab8992<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12080">issues.joomla.org/joomla-cms/12080</a>.</sub>

**Comment by wojsmol** [comment]
cc @JoomliC Please do a code reciew :smile:

**Comment by zero-24** [comment]
this is now moved to be against 3.7.x

**Pull request closed without merging**